webnode/src/components/consent-overlay/button-group/index.tsx

```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import color from "colors";
+import Radium from 'radium';
+
+interface ButtonProps {
+  onClick: () => void;
+  children: any;
+  kind: string;
+}
+
+interface ButtonState {
+}
+
+class Btn extends React.Component<ButtonProps, ButtonState> {
+  render() {
+    const style = {
+      base: {
+        cursor: "pointer",
+        borderRadius: 12,
+        outline: "none",
+        padding: 0,
+        fontFamily: "Poppins",
+        fontSize: 23,
+        fontWeight: 600,
+        minWidth: 300,
+        width: "100%",
+        height: 50,
+        margin: 15,
+        "@media (max-width: 1200px)": {
+          fontSize: 16,
+          margin: 5,
+          borderRadius: 8,
+          height: 40,
+          minWidth: 180
+        }
+      }
+    };
+    const { onClick, children } = this.props;
+    // TODO base[kind]
+    return (
+      <button onClick={onClick} style={style.base}>
+        {children}
+      </button>
+    );
+  }
+}
+
+const Button = Radium(Btn);
+
+interface ButtonGroupProps {
+  giveConsent: () => void;
+  denyConsent: () => void;
+}
+
+interface ButtonGroupState {
+}
+
+class ButtonGroup extends React.Component<ButtonGroupProps, ButtonGroupState> {
```

Comment:
@EdmundMai @AaronVasquez do you know if we should avoid two classes in a single component? These ones seem related but i have not seen this before.

Reply:
i think i saw this pattern when i was looking up examples. what does the `<...>` in `React.Component<ButtonGroupProps, ButtonGroupState>` do?

Reply:
Its casting, we have to define the shape of our props or we get TS errors.

Reply:
I dont know what you mean, I keeped what we had